projects/plugins/jetpack/modules/widgets/simple-payments.php

```diff
@@ -66,6 +66,7 @@ function __construct() {
 				array(
 					'classname'                   => 'jetpack-simple-payments',
 					'description'                 => __( 'Add a Pay with PayPal button as a Widget.', 'jetpack' ),
+					'show_instance_in_rest'       => true,
```

Comment:
I _think_ that unless we are planning to add a transform from the "Pay with PayPal" widget into the "Pay with PayPal" block, this line isn't necessary.

But maybe you are planning to add that transform or I am missing something. 🤔

Reply:
> maybe you are planning to add that transform 

I think that would be nice, in that PR or in a follow-up one!